usr/share/rear/conf/default.conf

```diff
@@ -2807,6 +2807,11 @@ BEXTRACT_DEVICE=
 # Leave unset, if you have only one fileset for your client (this is almost always the case)
 # E.g. BAREOS_FILESET=Full
 # BAREOS_FILESET=
+#
+# Sometimes the restore client is a different machine then the backup is taken from.
+# In this cases you can specify the restore client by BAREOS_RESTORE_CLIENT.
+# In most cases, leave this unset.
+# BAREOS_RESTORE_CLIENT=
 ####
```

Comment:
`BAREOS_DIRECTOR_ADDRESS` seems to be missing? Please ensure that all variables that a user **can** set are also documented here.

Reply:
No longer needed and therefore removed.